src/lang/modifyAst/addFillet.test.ts

```diff
@@ -313,3 +318,82 @@ const extrude001 = extrude(-5, sketch001)
     expect(edges).toEqual([])
   })
 })
+
+describe('Testing button states', () => {
+  const runButtonStateTest = async (
+    code: string,
+    segmentSnippet: string,
+    expectedState: boolean
+  ) => {
+    // ast
+    const astOrError = parse(code)
+    if (astOrError instanceof Error) {
+      return new Error('AST not found')
+    }
```

Comment:
Nothing wrong with this, so I'm not coercing you to change it, but the standard way we've been handling this is 
```suggestion
    if (err(astOrError)) {
      return new Error('AST not found')
    }
```

Reply:
thanks, pushed